### ISSUE TITLE
number of epochs

### ISSUE THREAD

**Issue by roimehrez** [CQFIO/PhotographicImageSynthesis#15]
You write in the paper that you rescale {lambda_l} after 100 epochs. 
1. What is the total number of epochs you use? 
2. Do you rescale the values such that all loss P0 to P5 will have the same expected value?

Thanks

**Comment by LinoNetwork** [comment]
1. I use 200 epochs for 256p, and fine-tune 20 epochs for 512p and 5 epochs for 1024p.

2. Yes, the main purpose is to let P0-P5 have similar contributions to the loss function.

**Comment by CQFIO** [comment]
1. I use 200 epochs for 256p, and fine-tune 20 epochs for 512p and 5 epochs for 1024p.

2. Yes, the main purpose is to let P0-P5 have similar contributions to the loss function.
